timed/reports/views.py

```diff
@@ -253,6 +260,17 @@ def _create_workreport(self, from_date, to_date, today, project, reports,
         table[13 + len(reports), 2].formula = 'of:=SUM(B13:B{0})'.format(
             str(13 + len(reports) - 1))
 
+        pos = 18 + len(reports)
+        for task in tasks.items():
```

Comment:
better:
`for task_name, task_total_hours in tasks.items()`

easier to read

Reply:
Done